docs/guides/recipe.rst

```diff
@@ -94,6 +115,178 @@ DAG Tasks
 As mentioned above, a Task can itself be a DAG. This both allows comlpex graphs
 to be reused inside other graphs and allows isolating related Tasks into a logical unit.
 
+Add a new file at the path ``flow/dag.yaml``. Inside that file, place the following snippet:
```

Comment:
Did you mean `flow/main.yaml`?

Reply:
I was thinking that the documentation example should make use of both, a `dag.yaml` that is referenced in the `main.yaml` to show how a nested DAG can be consumed in a Recipe.

Reply:
Ah right I get you. I would call it `sub-dag` or something that gives the idea that it is not the main entrypoint of the recipe. Thoughts @mostaphaRoudsari ?

Reply:
I'm going to put an example here to iterate over and energy simulation and we can call it based on the name of it's functionality. I'll do a first pass and you can keep it if you like it.